src/js/components/addData/AddDataMeta.jsx

```diff
@@ -118,10 +127,42 @@ export default class AddDataMeta extends React.Component {
         }
     }
 
-    submitMetadata(){
-        this.props.updateMetaData(this.state);
-    }
-
+    submitMetadata(e){
+        let agency = this.state.agency, 
+            endDate = this.state.endDate,
+            dateType = this.state.dateType,
+            year = endDate.substr(3),
+            quarter = endDate.substr(0, 2);  
+        switch (quarter) {
+            case '12': quarter = '3';
```

Comment:
also, the switch is more code than you need. You should be able to do
`quarter = (parseInt(quarter)%12) + 3;`
```
if (quarter == 3){
        year = parseInt(year)+1;
        //do you need to turn this back into a string?
}
```